ext/opentelemetry-exporter-cloud-trace/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -288,20 +292,32 @@ def _extract_events(events: Sequence[Event]) -> ProtoSpan.TimeEvents:
 
 
 def _extract_attributes(
-    attrs: types.Attributes, num_attrs_limit: int
+    attrs: types.Attributes,
+    num_attrs_limit: int,
+    add_agent_attr: bool = False,
 ) -> ProtoSpan.Attributes:
     """Convert span.attributes to dict."""
     attributes_dict = BoundedDict(num_attrs_limit)
-
+    invalid_value_dropped_count = 0
```

Comment:
What's the motivation for this change? Does it need a test?

Reply:
Uh this change is from another PR. I think this is just git being weird, lemme try and rebase.